source/dev-setup.rst

```diff
@@ -78,35 +77,24 @@ to start it running immediately::
 Python
 ''''''
 
-Different projects require different versions of Python. Some projects may
-run on a wider range of Python versions and some may depend on very
-specific versions.
+Pyenv via brew
 
-It is recommended Mac users install the `pythonz` tool, which automates the
-download, install, and configuration of specific versions of Python. It is
-capable of installing either official versions of Python (from python.org)
-or alternate versions like PyPy, Stackless, and Jython.
+    brew install pyenv
 
-Pythonz will install everything into a hidden `.pythonz/` directory in your
-home directory, and will not modify any paths by default.
+Different projects require different versions of Python. Some older projects will still be using
+Python 3.7, and newer projects are eiter on 3.5 or 3.6, so we'll install the latest versions of all
+three.
 
-You can install Pythonz easily::
+    pyenv install $(pyenv install -l | grep " 2.7" | tail -n 1)
+    pyenv install $(pyenv install -l | grep " 3.5" | tail -n 1)
+    pyenv install $(pyenv install -l | grep " 3.6" | tail -n 1)
 
-    curl -kL https://raw.github.com/saghul/pythonz/master/pythonz-install | bash
-    echo "[[ -s $HOME/.pythonz/etc/bashrc ]] && source $HOME/.pythonz/etc/bashrc" >> ~/.bash_profile
-    source $HOME/.pythonz/etc/bashrc
+And link them so they'll be in your path.
 
-or learn more at the Pythonz github page:
+    ln -s ~/.pyenv/versions/$(pyenv install -l | grep " 2.7" | tail -n 1 | strip-indent)/bin/python /usr/local/bin/python2.7
```

Comment:
These commands do not work for me. I get a "strip-indent" command not found error message.